src/track/trackmetadatataglib.cpp

```diff
@@ -838,6 +838,29 @@ void readTrackMetadataFromID3v2Tag(TrackMetadata* pTrackMetadata,
         parseTrackPeak(pTrackMetadata,
                 toQString(pTrackPeakFrame->fieldList()[1]));
     }
+    TagLib::ID3v2::FrameList ratingFrame = tag.frameListMap()["POPM"];
+    int rating = 0;
+    if(!ratingFrame.isEmpty()) {
+        // RatingString "traktor@native-instruments.de rating=255 counter=2"
+
+        QString sRating = TStringToQString(ratingFrame.front()->toString());
+        sRating = sRating.section("=",1,2).left(3);
+        float fRating = sRating.toInt();
+        rating = ceil(fRating/51);
```

Comment:
Since we want to write back the metadadata one day, it is Important not to loose information during the round trip in Mixxx. So we should do the mapping to stars only when we display the data. 

Here, I think a data structure like this will help:
```
class Popularity {
  public:
      unsigned char rating; 
      double counter;
}
...
QMap<QString, Popularity> m_popularity 
...
TrackMetadata::addPoplarity(const QString& email, const Popularity& popularity) {
    m_popularity.insert(email, popularity) 
}
```